recipe/meta.yaml

```
@@ -28,19 +37,28 @@ source:
     folder: vendor/fmt
 
 build:
-  number: 0
+  number: {{ build }}
   skip: true  # [win]
   skip: true  # [py<35]
```

```suggestion
  skip: true  # [py<36]
```
I guess the setup.py takes care of the minimum python versions but should this be now set to 3.6 after [PR 86](https://github.com/openmc-dev/openmc/pull/1860) was merged into openmc